WebApp/app.go

```diff
@@ -323,8 +323,10 @@ func main() {
 		}
 	})
 
-	logged.StaticFile("/", "./website/homepage.html")
-	logged.Static("/static", "./website")
+	logged.StaticFile("/", "./homepage/boardsite/dist/index.html")
+	logged.StaticFile("/image2vector (2).svg", "./homepage/boardsite/dist/image2vector (2).svg")
+	logged.StaticFile("/vite.svg", "./homepage/boardsite/dist/vite.svg")
```

Comment:
assuming we are not leaving the vite logo on our page at all, you can just remove this file and stop serving it